AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginAuthIntegrationTests/MultiAuth/README.md

```diff
@@ -7,7 +7,10 @@
 
 1. create a `schema.graphql` file with the content of  `MultiAuth/schema.graphql`
 
-2. `amplify init`
+2. `amplify init`. Make sure your amplify CLI version produces ClI.json containing
+```
+"usesubfordefaultidentityclaim": true
```

Comment:
Will `usesubfordefaultidentityclaim` be true by default once the CLI changes are released? If so, we can just update the CLI version used for provisioning this test insteaed of having the CLI.json updates in this step